### PR TITLE
Fix Payload Passthrough in the Serverless Plugin

### PR DESCRIPTION
**Pull request by erichaberkorn**
### Description
Renamed the payload passthrough key from "passhthrough" to "passthrough".

### CODE REVIEW

File: .changelog/13607.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```
Comment (lkysow) [inline .changelog/13607.txt:1]: this should be release-note:bug right?

Comment (erichaberkorn) [reply .changelog/13607.txt:1]: Nice  catch

**Review by lkysow — COMMENTED** [review]
Should our tests run through both passthrough false and passthrough true?

**Review by lkysow — APPROVED** [review]
With comments resolved from ent PR

**Pull request merged**